csv_datasets/ldbc/lists/edges-sf-003-half_001.txt

```diff
@@ -10,40 +10,40 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
 :START_ID(TagClass)|:END_ID(TagClass)|:TYPE
 sf-003/static/tagclass_isSubclassOf_tagclass/part-00000-3a05ed0e-440f-4389-bdd2-7ecabccdedc7-c000.csv.split_1
 
-creationDate:DATETIME|:START_ID(Comment)|:END_ID(Person)|:TYPE
+creationDate:IGNORE|:START_ID(Comment)|:END_ID(Person)|:TYPE
```

Comment:
I do not follow; why do we ignore in this case?

Reply:
Edge type hasCreator (that goes from node type Comment to node type Person has no creationDate property in the schema). I'm updating the PR description, hopefully that will shed more light on the issue

Reply:
The description now is very good.